src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrLookupIdsInDomainResponse.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2017, Rapid7, Inc.
+ *
+ * License: BSD-3-clause
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * * Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright
+ * notice, this list of conditions and the following disclaimer in the
+ * documentation and/or other materials provided with the distribution.
+ *
+ * * Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software
+ * without specific prior written permission.
+ */
+package com.rapid7.client.dcerpc.mssamr.messages;
+
+import java.io.IOException;
+import java.util.List;
+import com.rapid7.client.dcerpc.io.PacketInput;
+import com.rapid7.client.dcerpc.messages.RequestResponse;
+import com.rapid7.client.dcerpc.mssamr.objects.SAMPRReturnedUStringArray;
+import com.rapid7.client.dcerpc.mssamr.objects.SAMPRULongArray;
+import com.rapid7.client.dcerpc.objects.RPCUnicodeString.NonNullTerminated;
+
+/**
+ *
+ */
+public class SamrLookupIdsInDomainResponse extends RequestResponse {
+    private SAMPRReturnedUStringArray names = new SAMPRReturnedUStringArray();
+    private SAMPRULongArray uses = new SAMPRULongArray();
+
+    @Override
+    public void unmarshalResponse(PacketInput in) throws IOException {
```

Comment:
I'd suggest initialization `names` and `uses` here instead of on object creation. Following that convection makes all of our RPC layer objects reusable